Add support data for CSSPseudoElement

lpd-au opened the pull request:
Add support data for [`CSSPseudoElement`](https://developer.mozilla.org/en-US/docs/Web/API/CSSPseudoElement).

#### Firefox ####

Current status is flagged off for stable but present in Nightly.

* First implementation https://bugzilla.mozilla.org/show_bug.cgi?id=1174575
* Pref rename https://bugzilla.mozilla.org/show_bug.cgi?id=1471814

#### Chromium ####

Not present in Canary. See also https://bugs.chromium.org/p/chromium/issues/detail?id=813902#c5

#### Edge ####

Web Animations API is unsupported entirely by EdgeHTML.

#### Safari ####

Not present in iOS v12.3.1 (Safari 12.1.1) with both `Web Animations` and `CSS Animations via Web Animations` experiments turned on.

Probably not present in MacOS Safari 12.1 but may be present in Safari 13 technological preview (unsure). Possibly relevant patch: https://bugs.webkit.org/show_bug.cgi?id=186518

lpd-au commented:
@jpmedley
> This shipped in Chrome 70: https://storage.googleapis.com/chromium-find-releases-static/b82.html#b822dab24251df3891739e3cedb37dab519ebff7

No it didn't, That is the same commit message I linked to in the OP and as it says: _"Chromium still don't
support CSSPseudoElement"_.

vinyldarkscratch commented:
Can confirm that `CSSPseudoElement` is not present in the latest stable release either.  Sorry about that, Chrome looks good to go!  I'll be reviewing Safari desktop and taking a look into the latest Technology Preview shortly, and return to this PR.  :wink:

lpd-au commented:
I added an additional commit to add data for CSSPseudoElement's properties:

#### CSSPseudoElement.type ####

Has the same support as the parent interface.

#### CSSPseudoElement.element ####

Has the same support as the parent interface **except** Firefox used a different name (`parentElement`) until v67. Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1528883

#### Firefox Preference Renaming ###

I added a `version_removed` property to all Firefox 47 entries with the old obsolete, renamed preference. This seems to be consistent with other cases of renamed preferences.

ddbeck commented on api/CSSPseudoElement.json:
```diff
@@ -0,0 +1,307 @@
+          },
+          "firefox": [
+            {
+              "version_added": false
```
Otherwise, this looks great, but I did have one suggestion here (and repeated for the other Firefox entries): if the feature still exists behind a flag, then these `{ "version_added": false }` entries aren't necessary and can be removed.

lpd-au replied:
Done!
